omero/sysadmins/limitations.rst

```diff
@@ -43,15 +43,28 @@ user, you cannot make the group into a private group.
 File format support
 -------------------
 
+Large images
+^^^^^^^^^^^^
+
+When you import an image over a certain size, OMERO will generate a Pyramid of lower resolution
+images if it doesn't already exist in the file. The threshold size is configurable using
+:property:`omero.pixeldata.max_plane_height` and
+:property:`omero.pixeldata.max_plane_width` but set to 3192x3192 pixels by
+default. However, this process can be very resource-intensive for larger images, for example see
+`PixelData threads and pyramid generation issues <https://forum.image.sc/t/pixeldata-threads-and-pyramid-generation-issues/49794>`_.
+
+It is recommend that users avoid these issues by converting
```

Comment:
I would propose to add a first statement along the lines of 

"The OMERO pyramid generation process should be considered as deprecated and instead it is recommended..."